controllers/flowlogspipeline/flp_common_objects.go

```diff
@@ -527,6 +550,42 @@ func (b *builder) addTransformStages(stage *config.PipelineBuilderStage) error {
 		enrichedStage.EncodePrometheus("prometheus", promEncode)
 	}
 
+	if len(topPromMetrics) > 0 && b.info != nil && b.info.AvailableAPIs != nil && b.info.AvailableAPIs.HasSvcMonitor() {
```

Comment:
I don't think this should be tied to the ServiceMonitor API: exposing prometheus metrics should be possible even without it. For instance, non-openshift users may have their own prometheus scraping metrics, that doesn't use the prometheus operator hence no ServiceMonitor API

Reply:
I guess it was part of previous code before rebase.

Done https://github.com/netobserv/network-observability-operator/pull/391/commits/4175cfacf2405ef43039395df1ebe8583968cde1